clang/lib/CIR/CodeGen/CIRGenStmt.cpp

```diff
@@ -253,6 +253,7 @@ mlir::LogicalResult CIRGenFunction::emitSimpleStmt(const Stmt *s,
   case Stmt::NullStmtClass:
     break;
   case Stmt::CaseStmtClass:
+  case Stmt::DefaultStmtClass:
```

Comment:
What does 'top level' switch case mean here?  I realize it is pre-existing, but trying to grok what is going on here